pkg/cmd/server/server.go

```diff
@@ -158,6 +159,8 @@ func NewCommand(f client.Factory) *cobra.Command {
 			storeValidationFrequency:       defaultStoreValidationFrequency,
 			podVolumeOperationTimeout:      defaultPodVolumeOperationTimeout,
 			restoreResourcePriorities:      defaultRestorePriorities,
+			restoreOnly:                    false,
```

Comment:
If `restoreOnly` is default not set， the value would be false.
So do we need add this line?